drivers/input/keyboard/brain-kbd-i2c.c

```diff
@@ -101,38 +143,34 @@ static irqreturn_t bk_i2c_irq_handler(int irq, void *devid)
 	n = raw >> 8;
 	k1 = raw & 0xff;
 
+	dev_dbg(&kbd->cli->dev, "kbd-raw0[%02X,%02X]\n",n,k1);
 	if (k1 == 0x00) {
 		dev_dbg(&kbd->cli->dev,
 			"interrupted but no key press was found\n");
 		goto err;
 	}
 
-	if (n == 1) {
-		if (detect_key(kbd, k1)) {
-			goto done;
-		} else {
-			dev_dbg(&kbd->cli->dev,
-				"unknown key was pressed: %02x\n", (k1 & 0x3f));
-			goto err;
-		}
+	if(n<1) goto done;
+	if (!detect_key(kbd, k1)) {
+		dev_dbg(&kbd->cli->dev,
+			"unknown key was pressed: %02x\n", (k1 & 0x3f));
+		goto err;
 	}
+	if( n<2) goto done;
```

Comment:
非常に事故りやすいので、中括弧のない if は使用しないでください。
```suggestion
	if(n < 2) {
		goto done;
	}
```